### DISCUSSION TITLE
建议自动下载

### DISCUSSION THREAD

**Discussion by ivysrono** [the1812/Bilibili-Evolved#3149] [功能建议 / Ideas]
这样后台打开标签页，自动调用默认设置（如Aria2）下载，甚至可以设置下载完成后自动关闭，这样对于完全无需看评论、弹幕的人来说要稳定快捷很多。

如果这个需求太小众或者众口难调，能否给个指引，自己写脚本实现？

**Comment by the1812** [comment]
把弹幕关了, 不往下翻评论就行了

**Comment by ivysrono** [comment]
在线看会卡

**Reply by the1812** [reply]
那你自己写脚本吧, 我的源代码可以给你参考:
- 下载地址 API: registry/lib/components/video/download/apis/dash.ts
- aria2 RPC 输出: registry/lib/plugins/video/download/aria2-output/aria2-rpc.ts
